### PR TITLE
UBERF-7354 File edit page

### PR DESCRIPTION
**Pull request by aonnikov**
This pull requests adds a separate page for files instead of using preview.

Screenshot 2024-06-21 at 11 05 23

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2Njc0ZmMyODg3NDE2MjNkZGU3YjQ2NWYiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.K8WyMx9LlPgokbPtdWyPJp7FpUVRhvvlXza4udPYzL0">Huly&reg;: <b>UBERF-7357</b></a></sub>

### CODE REVIEW

File: packages/core/lang/ru.json [hunk 1]
```diff
@@ -61,6 +61,7 @@
     "ArchiveSpaceDescription": "Дает пользователям разрешение архивировать пространство",
     "AutoJoin": "Автоприсоединение",
     "AutoJoinDescr": "Автоматически присоединять новых сотрудников к этому пространству",
-    "BlobSize": "Размер"
```
Comment (haiodo) [inline packages/core/lang/ru.json:64]: Please add zh,fr languages

File: packages/ui/src/utils.ts [hunk 2]
```diff
@@ -78,6 +78,21 @@ export function floorFractionDigits (n: number | string, amount: number): number
   return Number(Number(n).toFixed(amount))
 }
 
+/**
+ * @public
+ */
+export function humanReadableFileSize (size: number, base: 2 | 10 = 10, fractionDigits: number = 2): string {
+  const units =
+    base === 10
+      ? ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
```
Comment (haiodo) [inline packages/ui/src/utils.ts:87]: Internationalisation?

**Review by haiodo — DISMISSED** [review]

File: packages/ui/src/utils.ts (hunk 2 above)
Comment (aonnikov) [reply packages/ui/src/utils.ts:87]: unit names are international and don't require translation

File: packages/core/lang/ru.json (hunk 1 above)
Comment (aonnikov) [reply packages/core/lang/ru.json:64]: done

**Review by haiodo — APPROVED** [review]

**Pull request merged**